_posts/en/newsletters/2020-01-08-newsletter.md

```diff
@@ -0,0 +1,96 @@
+regular section about notable changes to popular Bitcoin infrastructure
+projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Final week of organized taproot review:** December 17th was the
+  final scheduled [meeting][taproot meeting] of the taproot review
+  group.  Pieter Wuille posted the [slides][wuille slides] from a
+  presentation he gave summarizing progress, including text indicating
+  he thought the proposal was "nearly ready".  Wuille also proposed a
+  [minor change][wuille suggestion] to the tapleaf versioning.  Also
+  briefly mentioned was the [discussion][zmn post] started by ZmnSCPxj
+  on the Lightning-Dev mailing list about how precisely taproot could be
+  integrated with LN to provide improved privacy and scalability.
+
+- **Coinjoins without equal value inputs or outputs:** Adam Ficsor
+  (nopara73) started a [discussion][ficsor non-equal] on the Bitcoin-Dev
+  mailing list about two previously-published papers ([1][cashfusion],
+  [2][knapsack]) describing coinjoins that didn't use either equal-value
+  inputs or outputs.  Previous attempts at non-equal mixes were [easy to
+  compromise][coinjoin sudoku], but if an improved method was found, it
+  could significantly improve the privacy of coinjoins by making their
+  transactions look like [payment batching][].  This seemed especially
+  relevant after reports that a popular exchange was investigating users
+  participating in the chaumian-style coinjoins created by Wasabi
+  Wallet.  Although several ideas were discussed, we think the [summary][ontivero summary]
+  by Lucas Ontivero captures the essence of the overall conclusion: "In
+  summary, unequal inputs/outputs coinjoins [knapsack][] is the best we
+  have at the moment ([but] it is not as effective as equal-outputs
+  transactions)."
+
+- **Encoded descriptors:** Chris Belcher [asked][belcher descriptors]
+  for feedback from the Bitcoin-Dev mailing list about base64 encoding
+  [output script descriptors][topic descriptors] so that they're easier
+  to copy and paste (and also so that regular users aren't exposed to
+  their code-like syntax).  At least one reply was opposed to the idea
+  and other replies which did support the idea each proposed a different
+  encoding format.  The discussion to date did not come to any
+  clear conclusion.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #3351][] extends the `invoice` RPC with a new
+  `exposeprivatechannels` parameter that allows the user to request
+  the addition of route hints for private channels to a generated [BOLT11][]
```

Comment:
I like the current phrasing because (if I understand the change correctly), the route hints requested aren't guaranteed to be added to the invoice---they're only added if the daemon considers the "channels to be usable"  (e.g. they have incoming capacity).